File: python/sum_pairs.py
def sum_pairs(array, number):
    ans = []
    for i in array:
        for j in array:
            if i + j == number and [j,i] not in ans:
                ans.append([i,j])
    if len(ans) == 0:
        return 'unable to find pairs'
    return ans

File: js/sumPairs.js
const sumPairs = function(array, number) {
    let answ = []
    for (const i of array) {
        for (const j of array) {
            const ji = [j,i]
            const ij = [i,j]
            // const test = answ.includes(ji);
            const newLocal = i + j === number;
            if (newLocal) {
                answ.push(ij)
            }
        }
    }
    let answer = answ.filter([a,b] => )
    console.log(answ)
    if (answ.length == 0){
        return 'unable to find pairs'
    }
    
    return answ

};

// export function sumPairs()
console.log(sumPairs([1,2,3,4,5], 3))